Formula/cdogs-sdl.rb

```diff
@@ -24,7 +24,7 @@ def install
     system "make"
     prefix.install "src/cdogs-sdl.app"
     bin.write_exec_script "#{prefix}/cdogs-sdl.app/Contents/MacOS/cdogs-sdl"
-    pkgshare.install ["data", "dogfights", "graphics", "missions", "music", "sounds"]
+    pkgshare.install %w[data dogfights graphics missions music sounds]
```

Comment:
Note: this is only on longer string arrays.
@reitermarkus Why in this case? Your grepping argument doesn't stand up here so much.